src/config/errors.json

```diff
@@ -0,0 +1,32 @@
+{ 
```

Comment:
This file should be under errors folder not here

Reply:
moved the file in errors folder